twisted/conch/ssh/session.py

```diff
@@ -356,4 +356,4 @@ def packRequest_window_change(geometry):
     (rows, cols, xpixel, ypixel) = geometry
     return struct.pack('>4L', cols, rows, xpixel, ypixel)
 
-import connection
+from twisted.conch.ssh import connection
```

Comment:
The fix is ok, but since you are touching it, why not move it to the top ? :)

http://programmer.97things.oreilly.com/wiki/index.php/The_Boy_Scout_Rule